src/wl_window.c

```diff
@@ -429,14 +454,14 @@ void _glfwPlatformSetWindowSizeLimits(_GLFWwindow* window,
                                       int minwidth, int minheight,
                                       int maxwidth, int maxheight)
 {
-    // TODO
-    fprintf(stderr, "_glfwPlatformSetWindowSizeLimits not implemented yet\n");
+    // TODO: find out how to trigger a resize.
```

Comment:
Actually no, this function is specified as modifying the current width and height of the window when it gets called.  Now that I think about it, it should probably call `_glfwInputWindowSize()` to achieve the desired effect.